src/web/services/student-profile.service.ts

```diff
@@ -11,6 +11,7 @@ import { HttpRequestService } from './http-request.service';
   providedIn: 'root',
 })
 export class StudentProfileService {
+  profilePictureEndPoint: string = '/student/profilePic';
```

Comment:
Though personally I think it is OK to use string literal here. Depends on how you want to standardize this. Either use it for all API call or not use it.

Reply:
Ok I'll change it back and standardize it in a later PR